packages/components/src/lib/earnings/EarningsSummaryModal.tsx

```diff
@@ -77,8 +77,11 @@ const EarningsSummaryModal: FunctionComponent<
                 <Typography variant="subtitle1">Earnings</Typography>
                 <Typography variant="h4">
                   <Typography component="span" mr={ 0.5 } variant="subtitle2">
-                    (≈{ " " }
-                    { formatUsdAmount(unclaimedEarningsInUSD, { precision: 2 }) }){ " " }
+                    (
+                    { formatUsdAmount(unclaimedEarningsInUSD, {
+                      includeEstimateSymbol: true,
+                    }) }
+                    ){ " " }
```

Comment:
Do we know why these extra spaces are located everywhere with the currency format jsx? Existed before I added this in but I think one or two were added in for consistency I believe. Do we have issue removing them or are they in place for text formatting?

Reply:
Will look at creating a ticket for this one